src/Components/ProgressBar/ProgressBar.jsx

```diff
@@ -1 +1,34 @@
-// (SAM) - TO DO
\ No newline at end of file
+const ProgressBar = (props) => {
+  const { completed } = props;
+
+  const containerStyles = {
```

Comment:
[This is a good read](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx#:~:text=Naming%20Convention%20for%20Constants) for the naming convention of these `const` variables 👍